modules/external-api/src/main/java/org/opencastproject/external/endpoint/EventsEndpoint.java

```diff
@@ -170,7 +171,7 @@
 
 @Path("/")
 @Produces({ ApiMediaType.JSON, ApiMediaType.VERSION_1_0_0, ApiMediaType.VERSION_1_1_0, ApiMediaType.VERSION_1_2_0,
-            ApiMediaType.VERSION_1_3_0, ApiMediaType.VERSION_1_4_0, ApiMediaType.VERSION_1_5_0 })
+            ApiMediaType.VERSION_1_3_0, ApiMediaType.VERSION_1_4_0, ApiMediaType.VERSION_1_5_0, ApiMediaType.VERSION_1_6_0 })
```

Comment:
I think you missed some @Produces for methods, for example for getEventScheduling(). Some methods can be available for less versions than the rest, mostly when they were added later. Sorry. :D (I can show you my commit later.)

Reply:
Thanks for the review @KatrinIhler. I added now hopefully all missing @produces. It seems there was also one missing for 1.5.0 in the statistics endpoint.

Reply:
Yeah, I found that one as well. :D